src/libraries/System.Drawing.Primitives/tests/RectangleTests.cs

```diff
@@ -165,6 +165,7 @@ public static void GetHashCodeTest()
         [InlineData(float.MinValue, float.MaxValue, float.MinValue, float.MaxValue)]
         [InlineData(0, 0, 0, 0)]
         [ActiveIssue("https://github.com/dotnet/runtime/issues/60585", TestPlatforms.iOS | TestPlatforms.tvOS)]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/83394", TestPlatforms.Browser)]
```

Comment:
this fails on AOT only though. We can use `typeof(PlatformDetection), nameof(PlatformDetection.IsBrowser), name    of(PlatformDetection.IsMonoAOT))]`